nixos/modules/services/misc/persistent-evdev.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.persistent-evdev;
+  configFile = pkgs.writeText "persistent-evdev-config.json" ''
```

Comment:
maybe use `pkgs.formats.json` to generate this instead. the json format pretty-prints the resulting text automatically.